recipes/cuda-histogram/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "cuda_histogram" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a3672c474dd094e2fefd364757d8fb6aa016af6e2ffee356e2a9a78c88f58690
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: True  # [win or osx or py<38]
```

FWIW conda-forge is on Python 3.9+. So we can probably simplify this like so

```suggestion
  skip: True  # [win or osx]
```